src/docs/product/issues/issue-details/performance-issues/index.mdx

```diff
@@ -86,6 +87,10 @@ The following is a list of available performance issues:
     </tr>
 </table>
 
+<Note>
+* Currently only available to Early Adopters.
+</Note>
+
```

Comment:
Would be awesome if you could also add a link to the page in the Early Adopter section: https://docs.sentry.io/product/accounts/early-adopter/ <3